stdlib/LinearAlgebra/src/cholesky.jl

```diff
@@ -38,10 +38,10 @@ struct Cholesky{T,S<:AbstractMatrix} <: Factorization{T}
         new(factors, uplo, info)
     end
 end
-Cholesky(A::AbstractMatrix{T}, uplo::Symbol, info::BlasInt) where {T} =
-    Cholesky{T,typeof(A)}(A, char_uplo(uplo), info)
-Cholesky(A::AbstractMatrix{T}, uplo::AbstractChar, info::BlasInt) where {T} =
-    Cholesky{T,typeof(A)}(A, uplo, info)
+Cholesky(A::AbstractMatrix{T}, uplo::Symbol, info::Integer) where {T} =
+    Cholesky{T,typeof(A)}(A, char_uplo(uplo), BlasInt(info))
```

Comment:
Actually, I shouldn't even be necessary to convert explicitly to `BlasInt`. The conversion will happen implicitly if you just pass `info` and it is slightly cleaner.

Reply:
is that true of the `Vector` too?

Reply:
Yes. It should also be the case the `Vector.